test/UploadedFileTest.php

```diff
@@ -146,13 +146,14 @@ public function testMovesFileToDesignatedPath()
     {
         $stream = new Stream('php://temp', 'wb+');
         $stream->write('Foo bar!');
+        $originalContents = $stream->__toString();
```

Comment:
This leads to the stream being consumed before being passed to `UploadedFile` - this test should be left alone